test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/TestAutoCreateSharedArchiveUpgrade.java

```diff
@@ -134,11 +164,75 @@ static void assertCreatedJSA(OutputAnalyzer output) {
         output.shouldContain("Dumping shared data to file");
     }
 
-    static void assertJSAVersionMismatch(OutputAnalyzer output) {
-        output.shouldContain("does not match the required version");
-    }
-
     static void assertUsedJSA(OutputAnalyzer output) {
         output.shouldContain("Mapped dynamic region #0");
     }
+
+    // Fetch JDK artifact depending on platform
+    // If the artifact cannot be found, default to the test.boot.jdk property
```

Comment:
This comment about test.boot.jdk is no longer valid. If the artifact cannot be found, RuntimeException is thrown.